pkg/tid/cli/display/table.go

```diff
@@ -76,6 +77,15 @@ func WriteTimesheetsTable(sheets []types.Timesheet, writer io.Writer) {
 	table.Render()
 }
 
+// getTimeInRightFormat prints the time using the format specified in config file.
+func getTimeInRightFormat(duration time.Duration, timeFormat string) string {
+	if (timeFormat == "decimal") {
```

Comment:
Aye, `gofmt` and `goimports` will sort this out. I'll pester Ioannis about it in the office though, with the new IntelliJ update it has built-in support finally.